Dapper NET40/SqlMapper.cs

```diff
@@ -3036,14 +3040,7 @@ internal static IList<LiteralToken> GetLiteralTokens(string sql)
             var matches = literalTokens.Matches(sql);
             var found = new HashSet<string>(StringComparer.Ordinal);
             List<LiteralToken> list = new List<LiteralToken>(matches.Count);
-            foreach(Match match in matches)

```

Comment:
replacing a perfectly good foreach loop with LINQ is *not* code improvement, and adds a whole host of additional overhead to the method; will reject